### ISSUE TITLE
Region Definitions

### ISSUE THREAD

**Issue by kdettman** [emapple/social#2]
How are the regions as stated in energy_usage.ipynb defined? Specifically, which states belong to which regions?

**Comment by emapple** [comment]
After some digging, I still haven't found an official answer. If you go to https://www.eia.gov/beta/electricity/gridmonitor/dashboard/electric_overview/US48/US48 and hover your mouse, you can see on the map what each region covers.

If this is sufficient, we can hard-code a list based on what we see on the map. The only issue is, some states are split. Illinois, for example, is mostly Midwest, but the Chicago area is Mid-Atlantic.

Let me know what you'd like to do.

**Comment by kdettman** [comment]
Yes, there are many half states in each region. Even Texas isn't all of Texas. Right now our data is divided at the state level so I think we should still focus on state level analyses. We have the capability to use more granular data, but that would involve a bit of retooling. For exploratory analysis I think we can stick to just including the weather data for a state that is substantially involved in the region in question. If this is not good enough to explain what we're seeing then we can discuss going more granular.

**Comment by emapple** [comment]
If we want energy data on <monthly timescales, I don't _think_ we can go more granular than regions. If you think we can, please link to the data source for it.

I'd say that most of the states are entirely in one EIA region or another, so we could if we wanted do some sort of averaging between state weather.

Alternately, we can focus on the several regions which are largely equivalent to states.
